cookbook/models/openai/chat/standalone.py

```diff
@@ -0,0 +1,13 @@
+"""
+This example shows how to use the OpenAIChat model as a standalone model.
+"""
+
+from agno.agent import Message
+from agno.models.openai import OpenAIChat
+from agno.models.response import ModelResponse
+
+model = OpenAIChat(id="gpt-4o")
+
+response: ModelResponse = model.invoke(messages=[Message(role="user", content="Hello, how are you?")])
+
+print(response.content)
```

Comment:
Woah that's sweet!

- Something like this cookbook should make it to the top of our docs. 
- `model.run(message: str)` and `model.print()` would be nice add-ons

Reply:
Do you think we should rename `invoke` to `run`? We can

Reply:
Yeah! sounds more clear to me